client/package.json

```diff
@@ -358,9 +358,9 @@
           "type": "object",
           "default": {
             "gobraToolsBasePath": {
-              "windows": "%APPDATA%\\Gobra\\",
-              "linux": "$HOME/.config/Gobra",
-              "mac": "$HOME/Library/Application Support/Gobra"
+              "windows": "%APPDATA%\\Roaming\\Code\\User\\globalStorage\\viper-admin.gobra-ide\\Local\\GobraTools",
+              "linux": "$HOME/.config/Code/User/globalStorage/viper-admin.gobra-ide/Local/GobraTools",
+              "mac": "$HOME/Library/Application Support/Code/User/globalStorage/viper-admin.gobra-ide/Local/GobraTools"
```

Comment:
did you test whether these paths work fine in windows and linux? If not, I can test it

Reply:
Not yet. Expanding `$HOME` and `%APPDATA%` is taken from Viper-IDE. Note however that these are "suggested" paths as no file will be located there. The idea is that one could place a "Local" folder with the local version of the GobraTools there. Nevertheless, would be great if you could give it a quick try on the other platforms